src/aspire/source/simulation.py

```diff
@@ -245,6 +248,115 @@ def _apply_sim_filters(self, im, indices):
             self.filter_indices[indices],
         )
 
+    def estimate_signal_mean(self, sample_n=100):
+        """
+        Estimate the signal mean of `sample_n` projections.
+
+        :param sample_n: Number of projections used for estimate.
+        :returns: Estimated signal power.
+        """
+        # Note, for simulation we are assuming `sample_n` is random
+        estimated_mu = np.mean(self.projections[:sample_n].asnumpy())
+        logger.info(f"Estimated signal mu {estimated_mu}")
+        return estimated_mu
+
+    def estimate_signal_var(self, sample_n=100):
+        """
+        Estimate the signal the variance of `sample_n` projections.
+
+        :param sample_n: Number of projections used for estimate.
+        :returns: Estimated signal power.
```

Comment:
The estimate mean and estimate variance docstrings both say they return the power..

Reply:
yes they are both typos. fixed.